0_portfolio_input_check_functions.R

```diff
@@ -835,7 +849,7 @@ create_id_columns <- function(portfolio, portfolio_type){
 # FINAL SCRIPTS
 get_and_clean_currency_data <- function(){
   
-  currencies <- readRDS("data/currencies.rda")
+  currencies <- read_file("data/currencies.rda")
```

Comment:
this will probably cause the masking error @Clare2D I would argue readRDS is safer for now and we should change the function name